lib/Smart/Args/TypeTiny.pm

```diff
@@ -319,12 +319,11 @@ For optimization calling subroutine in runtime type check, you can overwrite C<c
 
     {
         no warnings 'redefine';
-        sub Smart::Args::TypeTiny::check_rule {
-            my ($rule, $value, $exists, $name) = @_;
-            return $value;
-        }
+        *Smart::Args::TypeTiny::check_rule = \&Smart::Args::TypeTiny::Check::no_check_rule;
     }
 
+C<Smart::Args::TypeTiny::Check::no_check_rule> is a function without type checking, but settings such as default and optional work the same as C<check_rule>.
```

Comment:
Would you please document that if the type has coercion (such as the following) it will not work correctly?
https://github.com/akiym/Smart-Args-TypeTiny/blob/7017e54a481c0a79def050f3e7633a06efa28893/t/12_coercion.t#L5

Reply:
Thanks for your review. I fixed! https://github.com/akiym/Smart-Args-TypeTiny/pull/5/commits/3aae9dfde21b2b2c7bd4eff13193b90752297f5b